### ISSUE TITLE
[Question]: Is it possible that we can integrate with llamaindex?

### ISSUE THREAD

**Issue by allanwakes** [infiniflow/infinity#2272]
### Describe your problem

Infinity as an AI-native database, looks very promising.
Is there a plan to integrate with llamaindex, so we can benefit from using them together?

Thanks.

**Comment by yingfeng** [comment]
Yes, we have. But the integration requires some work and we are short of resources, so contributions are welcome !

**Comment by zpf121** [comment]
Close
